src/util/entities/User.ts

```diff
@@ -138,6 +139,9 @@ export class User extends BaseClass {
 	@Column({ select: false })
 	mfa_enabled: boolean = false; // if multi factor authentication is enabled
 
+	@Column({ select: false, default: false })
+	webauthn_enabled: boolean = false; // if webauthn multi factor authentication is enabled
```

Comment:
DB migrations?

Reply:
I thought we used sync and didnt need migrations?

Reply:
We use sync when a database doesn't exist, but otherwise use migrations. Sync causes data loss in some cases